.changeset/beige-penguins-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-common': major
+---
+
+Remove `map()`
```

Comment:
@josephjclark mogil and salesforce adaptor use this function, We should remove the export from those adaptor as well